api/src/main/resources/workbench-api.yaml

```diff
@@ -550,7 +550,9 @@ paths:
     get:
       tags:
       - user
-      description: Returns user information matching search terms
+      description: >
```

Comment:
No changes to the current endpoint.  Update the description for clarity and  make an internal change to make a later transition easier.  See UserServiceImpl and UserDao `findUsersBySearchString()` methods for details.